pkg/sourcereader/embedded.go

```diff
@@ -28,9 +28,10 @@ import (
 // hpc-toolkit/modules are not accessible at the package level.
 var ModuleFS BaseFS
 
-// BaseFS is an extension of the io.fs interface with the functionality needed
+// BaseFS is an extension of the fs.FS interface with the functionality needed
 // in CopyDirFromModules. Works with embed.FS and afero.FS
 type BaseFS interface {
+	fs.FS
```

Comment:
This looks like the only "real" change in this PR. The old comment appears to be wrong in 2 ways:

- it confuses `io/fs` and `io.fs`
- it doesn't appear to extend (embed) any interface

The use of `fs BaseFS` in function signatures is potentially confusing due to overlap with `io/fs`.  I suggest we fix this now by picking a different name for the variable. `basefs`, `bfs`?